src/signals/incident-management/containers/IncidentDetail/components/StatusForm/StatusForm.test.tsx

```diff
@@ -34,40 +34,15 @@ import {
   DEFAULT_TEXT_MAX_LENGTH,
   MELDING_CHECKBOX_DESCRIPTION,
 } from './constants'
+import type { StandardTextsResponse } from './StatusForm'
 import { PATCH_TYPE_STATUS } from '../../constants'
 import IncidentDetailContext from '../../context'
 import type { IncidentChild } from '../../types'
 
 jest.mock('shared/services/configuration/configuration')
 
 const incidentFixture = incidentJSON as unknown as Incident
-const defaultTexts = [
-  {
-    state: StatusCode.Ingepland,
-    templates: [
-      { title: 'Ingepland', text: 'Over 1 dag', is_active: true },
-      { title: 'Ingepland', text: 'Over 2 dagen', is_active: true },
-      { title: '', text: '', is_active: true },
-    ],
-  },
-  {
-    state: StatusCode.Afgehandeld,
-    templates: [
-      {
-        title: 'Niet opgelost',
-        text: 'Geen probleem gevonden',
-        is_active: false,
-      },
-      {
-        title: 'Niet opgelost',
-        text: 'Niet voor regio Amsterdam',
-        is_active: true,
-      },
-      { title: 'Opgelost', text: 'Lamp vervangen', is_active: true },
-      { title: 'Opgelost', text: 'Lantaarnpaal vervangen', is_active: true },
-    ],
-  },
-]
+let defaultTexts: any
```

Comment:
no type?

Reply:
That's on purpose. Since the texts have two different types (DefaultTexts and StandardTexts). This way we avoid TS warnings all over the place in the test.

Reply:
union type? don't really care might be neat